hack/machineset.sh

```diff
@@ -286,6 +303,33 @@ $(get_spec $infraID "" $provider)
 EOF
 }
 
+winver="2022"
+while getopts ":w:" opt; do
+  case ${opt} in
+    w ) # Windows Server version to use in the MachineSet. Defaults to 2022. Other option is 2019.
+      winver="$OPTARG"
+      if [[ ! "$winver" =~ 2019|2022$ ]]; then
+        echo "Invalid -w option $winver. Valid options are 2019 or 2022"
+        exit 1
+      fi
+      ;;
+    \? )
+      echo "Usage: $0 -w <2019/2022> apply/delete"
```

Comment:
Please also update the usage info comment at the top of the script

Reply:
Done.